src/main/dml/fenixedu-academic.dml

```diff
@@ -1348,6 +1348,7 @@ valueType org.fenixedu.academic.util.sibs.incomming.SibsIncommingPaymentFileDeta
                 }
 
             class accounting.events.insurance.InsuranceEvent extends accounting.events.AnnualEvent {
+                DateTime dueDate;
```

Comment:
Do we really need this in the dml/database? 
Couldn't we just calculate the dueDate as needed using getWhenOccured and numberOfDaysToCalculateDueDate?

Reply:
we need it since if somebody changes the PR then the past will change which is not what we want.